### PR TITLE
Wrapped user visible strings with i18n* calls

### PR DESCRIPTION
**Pull request by shalomRachapudi**

### CODE REVIEW

File: src/installscreen.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ InstallScreen::InstallScreen(PackageBackend *backend, QString *tmpFileName, QObj
     installLayout->setSpacing( 20 );
 
     foreach( QUrl iter, m_backend->repositories() ) {
-        QLabel *sourceLabel = new QLabel( QString( "Added Source: %1" ).arg( iter.toString() ) );
+        QLabel *sourceLabel = new QLabel( i18n("Added Source: %1", iter.toString()) );
```
Comment (antlarr) [inline src/installscreen.cpp]: I would prefer to use the more standard i18n("Added Source: %1").arg( iter.toString() )  form than passing the argument to the i18n call. The way you did it is ok, but I think it's not the usual way to do it. Do you have a strong opinion about using that form?

File: src/firstscreen.h [hunk 2]
```diff
@@ -13,6 +13,7 @@
 #include <QTextStream>
 #include <QUuid>
 #include <QSettings>
+#include <klocalizedstring.h>
```
Comment (antlarr) [inline src/firstscreen.h]: klocalizedstring.h is not used in firstscreen.h so it should be included in firstscreen.cpp, not here.

File: src/installscreen.cpp (hunk 1 above)
Comment (shalomRachapudi) [inline src/installscreen.cpp]: I don't. I just followed the KI18N Documentation

File: src/firstscreen.h (hunk 2 above)
Comment (shalomRachapudi) [inline src/firstscreen.h]: I suppose that's true for all .h files. Shall I change it then?

File: src/mainheader.cpp [hunk 3]
```diff
@@ -47,20 +47,20 @@ void MainHeader::changeStatusLabel( int repoCount, int packageCount )
...
     m_repoCount = repoCount;
 
-    m_statusLabel->setText( QString( "This installer will download and install %1 packages from %2 sources" ).arg( packageCount ).arg( repoCount ) );
+    m_statusLabel->setText( i18np("This installer will download and install %1 package from %2 source", "This installer will download and install %1 packages from %2 sources", packageCount, repoCount) );
 }
 
 void MainHeader::updateDetails( QString size )
 {
-    m_statusLabel->setText( QString( "This installer will download and install %1 packages from %2 sources totalling %3" ).arg( m_packageCount ).arg( m_repoCount ).arg( size ) );
+    m_statusLabel->setText( i18np("This installer will download and install %1 package from %2 source totalling %3", "This installer will download and install %1 packages from %2 sources totalling %3", m_packageCount, m_repoCount, size) );
```
Comment (antlarr) [inline src/mainheader.cpp]: Perhaps you could use here i18ncp to add a context like "3rd argument is the size of all packages" . That way translators won't doubt what "totalling %3" means (which I think is not clear without a context)

Comment (shalomRachapudi) [inline src/mainheader.cpp]: Okay. I will :)

File: src/packagedetails.cpp [hunk 4]
```diff
@@ -42,7 +42,7 @@ PackageDetails::PackageDetails(OCI::Package *package,int count, int packagecount
 
     QObject::connect( meta, SIGNAL( finished( QString,QString ) ), this, SLOT( dataChanged( QString,QString ) ) );
 
-    m_showDescription = new QLabel( QString( "<a href = %1>Show Details</a>" ).arg( count ) );
+     m_showDescription = new QLabel( i18n("<a href = %1 style=\"text-decoration:none;\">Show Details</a>", count) );
```
Comment (antlarr) [inline src/packagedetails.cpp]: QString( "<a href = %1>%2</a>" ).arg( count ).arg( i18n("Show Details") )
so translators don't have to mess (and maybe break by accident) the markup.
Also, the style change wasn't mentioned in the commit log, so maybe it should be left for another pull request.

File: src/firstscreen.h (hunk 2 above)
Comment (antlarr) [inline src/firstscreen.h]: Yes please

File: src/installscreen.cpp (hunk 1 above)
Comment (antlarr) [inline src/installscreen.cpp]: Yes, as I said, it's not wrong, but it's not common to use it like that afaik. Would you mind changing that to the usual form?

File: src/packagedetails.cpp (hunk 4 above)
Comment (shalomRachapudi) [inline src/packagedetails.cpp]: It is mentioned in the individual commit but not in the PR. I shall make the necessary changes

File: src/installscreen.cpp (hunk 1 above)
Comment (shalomRachapudi) [inline src/installscreen.cpp]: I don't mind changing them to the usual form. I'll do them ASAP

File: src/packagedetails.cpp (hunk 4 above)
Comment (antlarr) [inline src/packagedetails.cpp]: ah, sorry, I didn't notice that

File: src/mainheader.cpp [hunk 5]
```diff
@@ -47,20 +47,20 @@ void MainHeader::changeStatusLabel( int repoCount, int packageCount )
...
 void MainHeader::updateDetails( QString size )
 {
-    m_statusLabel->setText( QString( "This installer will download and install %1 packages from %2 sources totalling %3" ).arg( m_packageCount ).arg( m_repoCount ).arg( size ) );
+    m_statusLabel->setText( i18ncp( "Third argument is the total size of all packages", "This installer will download and install %1 package from %2 source totalling %3", "This installer will download and install %1 packages from %2 sources totalling %3", m_packageCount, m_repoCount, size) );
 }
 
 void MainHeader::installationStarted()
 {
-    m_statusLabel->setText( "Downloading and Installing Packages..." );
+    m_statusLabel->setText( i18ncp("Decides which sentence to use based on m_packageCount","Downloading and Installing Package...", "Downloading and Installing Packages...", m_packageCount) );
```
Comment (antlarr) [inline src/mainheader.cpp]: The translator doesn't know (nor should know) anything about m_packageCount. Maybe use something like "Depends on the number of packages" ?

File: src/summary.cpp [hunk 6]
```diff
@@ -18,17 +18,18 @@
...
 Summary::Summary(PackageBackend *backend, QString *tmpFileName, QObject *parent )
 {
     m_backend = backend;
     m_tmpFileName = tmpFileName;
     m_installationSummary = new QTextBrowser;
     m_installationSummary->addScrollBarWidget( new QScrollBar, Qt::AlignRight );
-    m_continue = new QPushButton( "Continue Installation ");
-    m_cancel = new QPushButton( "Cancel ");
+    m_continue = new QPushButton( i18n("Continue Installation ") );
+    m_cancel = new QPushButton( i18n("Cancel ") );
```
Comment (antlarr) [inline src/summary.cpp]: Why is there a space at the end of those two strings? I think that was wrong before too.

Comment (shalomRachapudi) [inline src/summary.cpp]: My bad. I thought of rectifying it but forgot - spaces were there even before I started working on this code :)

File: src/mainwindow.cpp [hunk 7]
```diff
@@ -83,7 +84,7 @@ MainWindow::MainWindow( const QString& filename, QString tmpFileName, bool fakeR
     mainLayout->addLayout( buttonLayout );
 
     setLayout( mainLayout );
-    setWindowTitle( "One Click Install" );
+    setWindowTitle( i18n("One Click Install") );
```
Comment (shalomRachapudi) [inline src/mainwindow.cpp]: Even I'm not sure.  It's up to you though. Few names cannot be translated for eg - YaST but OCI is made up of legit words.

File: src/mainheader.cpp (hunk 5 above)
Comment (shalomRachapudi) [inline src/mainheader.cpp]: Sure. Changing it
